### DISCUSSION TITLE
Change all date format

### DISCUSSION THREAD

**Discussion by warsemann** [filamentphp/filament#5452] [Help] [answered]
Hi, can i change all the  date render in admin filament forms and table ? for example 25, Oct. 1980 in place of oct. 25, 2022 ?

Thanks

![image](https://user-images.githubusercontent.com/77194719/212031165-43146704-cf89-4cc8-82be-ff3f70afe73b.png)

**Comment by danharrin** [comment, marked as answer]
In the forms config file I think

**Comment by warsemann** [comment]
Nice Thanks, i only published the admin config file
